cmd/workspace/apps/apps.go

```diff
@@ -920,28 +939,49 @@ func newUpdate() *cobra.Command {
 	cmd := &cobra.Command{}
 
 	var updateReq apps.UpdateAppRequest
+	updateReq.App = &apps.App{}
 	var updateJson flags.JsonFlag
 
 	// TODO: short flags
 	cmd.Flags().Var(&updateJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
-	cmd.Flags().StringVar(&updateReq.Description, "description", updateReq.Description, `The description of the app.`)
+	// TODO: complex arg: active_deployment
+	// TODO: complex arg: app_status
+	// TODO: complex arg: compute_status
+	cmd.Flags().StringVar(&updateReq.App.CreateTime, "create-time", updateReq.App.CreateTime, `The creation time of the app.`)
+	cmd.Flags().StringVar(&updateReq.App.Creator, "creator", updateReq.App.Creator, `The email of the user that created the app.`)
+	cmd.Flags().StringVar(&updateReq.App.DefaultSourceCodePath, "default-source-code-path", updateReq.App.DefaultSourceCodePath, `The default workspace file system path of the source code from which app deployment are created.`)
+	cmd.Flags().StringVar(&updateReq.App.Description, "description", updateReq.App.Description, `The description of the app.`)
+	// TODO: complex arg: pending_deployment
 	// TODO: array: resources
+	cmd.Flags().Int64Var(&updateReq.App.ServicePrincipalId, "service-principal-id", updateReq.App.ServicePrincipalId, ``)
+	cmd.Flags().StringVar(&updateReq.App.ServicePrincipalName, "service-principal-name", updateReq.App.ServicePrincipalName, ``)
+	cmd.Flags().StringVar(&updateReq.App.UpdateTime, "update-time", updateReq.App.UpdateTime, `The update time of the app.`)
+	cmd.Flags().StringVar(&updateReq.App.Updater, "updater", updateReq.App.Updater, `The email of the user that last updated the app.`)
+	cmd.Flags().StringVar(&updateReq.App.Url, "url", updateReq.App.Url, `The URL of the app once it is deployed.`)
 
-	cmd.Use = "update NAME"
+	cmd.Use = "update NAME NAME"
```

Comment:
Now NAME param is in both request body and path. @pietern any ideas how do we address this for CLI? It might as well be necessary to fix it in OpenAPI

Reply:
cc @renaudhartert-db